storm-server/src/main/java/org/apache/storm/LocalCluster.java

```diff
@@ -231,6 +233,10 @@ private LocalCluster(Builder builder) throws Exception {
             } else {
                 this.clusterState = builder.clusterState;
             }
+            if (!Time.isSimulating()) {
+                //Ensure Nimbus assigns topologies as quickly as possible
+                conf.put(DaemonConfig.NIMBUS_MONITOR_FREQ_SECS, 1);
```

Comment:
Is this change also useful for other cases (non-test) as well? If we are unsure about it, could we setup global config for non time simulating test and add this to there instead?

Reply:
I think this is still useful outside testing, since the topology will start faster, and topologies will usually be submitted right after bootup of the local Nimbus. Are we intending for people to use LocalCluster for cases other than testing though?

Reply:
By "testing" I mean both automatic tests and manual tests

Reply:
OK makes sense. Doesn't seem to be a big deal as I think about this again.